core/src/test/scala/org/ensime/indexer/FileWatcherSpec.scala

```diff
@@ -57,7 +57,7 @@ class FileWatcherSpec extends EnsimeSpec
 
   val maxWait = 20 seconds
 
-  "FileWatcher" should "detect added files" taggedAs (Retryable) in
+  "FileWatcher" should "detect added files" taggedAs (IgnoreOnTravis, IgnoreOnAppVeyor, Retryable) in
```

Comment:
We don't want to ignore these on windows or osx. The problem was rogue threads, but I fixed it.